report_layout_config/views/res_company.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+        <record id="view_company_form" model="ir.ui.view">
+            <field name="name">view.company.form</field>
+            <field name="model">res.company</field>
+            <field name="inherit_id" ref="base.view_company_form"/>
+            <field name="arch" type="xml">
+               <xpath expr="//field[@name='favicon']" position="after">
+                  <field name="full_header_img" widget="image"/>
```

Comment:
IMO these fields should be made available in the General Settings wizard under the Business Documents section and displayed only if `external_layout_images` is selected. (means you have to declare related fields on the `res.config.settings`)